.travis.yml

```
@@ -5,4 +5,6 @@ notifications:
 
 install: npm ci
 
-script: npm test
+script: 
+  - npm test
+  - npm lint
```

```suggestion
  - npm run lint
  - npm test
```

Lint's the fast one, so best to have it go first - fail fast!